### DISCUSSION TITLE
Search in different vaults without switching to them.

### DISCUSSION THREAD

**Discussion by Sinled** [chrisgrieser/shimmering-obsidian#189] [Ideas]
Do you think it would be possible to add such behaviour? I have several unrelated vaults that are used in equal proportions and it would be nice to have ability to directly search in vaults with separate hotkeys without manually toggling active vault. Or maybe I can just to duplicate alfred workflow?

**Comment by chrisgrieser** [comment]
No, unfortunately that is not possible. and I have no plans of implementing such a behavior, since it is quite complicated to implement (how to decide which files to prioritize in what way, etc.)

Usually, vaults in Obsidian are meant to be a "closed-off" unit anyway, if you find yourself working in multiple vaults in parallel often, you should rather consider merging your vaults into one big vault.

**Comment by Sinled** [comment]
For anyone interested, duplicating alfred workflow and removing everything from it besides search part worked for me.
